documentation/modules/configuring/con-maintenance-time-window-definition.adoc

```diff
@@ -6,13 +6,38 @@
 = Scheduling maintenance time windows
 
 [role="_abstract"]
-Schedule Cluster Operator certificate renewals for Kafka clusters to minimize impact on client applications.
-Use time windows in conjunction with the xref:con-certificate-renewal-str[renewal periods of the CA certificates created by the Cluster Operator] (`Kafka.spec.clusterCa.renewalDays` and `Kafka.spec.clientsCa.renewalDays`).
+Use maintenance time windows to control when the Cluster Operator performs certificate renewals and rolling restarts.
```

Comment:
This might be misinterpreted as if it applies to all rolling updates. So maybe this way it will be a bit more clear?

```suggestion
Use maintenance time windows to control when the Cluster Operator performs certificate renewals and the related rolling restarts.
```